common/eventEmitter.js

```diff
@@ -0,0 +1,33 @@
+import EventEmitter from "events";
+import fs from "fs";
+import { getCurrentDate } from "./formatDate.js";
+import { fileURLToPath } from "url";
+import path from 'path';
+
+const eventEmitter = new EventEmitter();
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const filePath = path.join(__dirname, "../filesUpload.log");

```

Comment:
I would recommend using code without the hardcoded stash. 
For example:
`path.join(__dirname, '..', 'filesUpload.log'); `

hardcoding the slash (/) in paths can become a problem, especially when your application runs on different operating systems.
On Linux/macOS: The forward slash (/) is used as a directory separator.
On Windows: The backslash (\) is used as the directory separator, though Windows can sometimes handle forward slashes too, but it's not guaranteed.

Reply:
changed